### PR TITLE
Add comment option in CREATE VIEW documentation

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Add comment option in `CREATE VIEW` documentation
https://github.com/trinodb/trino/blob/master/core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4#L86-L88

## Documentation

(x) Sufficient documentation is included in this PR.

## Release notes

(x) No release notes entries required.

### CODE REVIEW

**Review by mosabua — APPROVED** [review]
Concise and correct... but could we maybe add an example.

File: docs/src/main/sphinx/sql/create-view.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ Synopsis
 .. code-block:: text
 
     CREATE [ OR REPLACE ] VIEW view_name
+    [ COMMENT view_comment ]
```
Comment (raunaqmorarka) [inline docs/src/main/sphinx/sql/create-view.rst:11]: It would be nice to have an example query with COMMENT as well

Comment (ebyhr) [reply docs/src/main/sphinx/sql/create-view.rst:11]: Added an example.

File: docs/src/main/sphinx/sql/create-view.rst [hunk 2]
```diff
@@ -48,6 +49,12 @@ Create a simple view ``test`` over the ``orders`` table::
     SELECT orderkey, orderstatus, totalprice / 2 AS half
     FROM orders
 
+Create a view ``test_with_comment`` with a view comment::
+
+    CREATE VIEW test_with_comment COMMENT 'A view to keep track of orders.' AS
```
Comment (mosabua) [inline docs/src/main/sphinx/sql/create-view.rst:54]:
```
CREATE VIEW test_with_comment 
    COMMENT 'A view to keep track of orders.' 
AS
    SELECT orderkey, orderstatus, totalprice
    FROM orders
```

**Review by raunaqmorarka — APPROVED** [review]

File: docs/src/main/sphinx/sql/create-view.rst (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/sql/create-view.rst]: I would add the indents as well ..

Comment (mosabua) [reply docs/src/main/sphinx/sql/create-view.rst]: maybe with only two spaces..

Comment (ebyhr) [reply docs/src/main/sphinx/sql/create-view.rst]: While I like adding indents, there's no such consensus and `CTAS` example have no indents.

Comment (mosabua) [reply docs/src/main/sphinx/sql/create-view.rst]: CREATE MATERIALIZED VIEW does have the indents .. and CTAS and others should have as well. We are aiming to make it as readable as possible and attempt to follow the Mozilla guidelines - https://docs.telemetry.mozilla.org/concepts/sql_style.html

Comment (ebyhr) [reply docs/src/main/sphinx/sql/create-view.rst]: > We are aiming to make it as readable as possible and attempt to follow the Mozilla guidelines - https://docs.telemetry.mozilla.org/concepts/sql_style.html

Please document in DEVELOPMENT.md or somewhere.

Comment (ebyhr) [reply docs/src/main/sphinx/sql/create-view.rst]: I would merge as-is since indents only for comment example doesn't make sense.

**Pull request merged**